themes/bootstrap3/js/record.js

```diff
@@ -343,6 +393,8 @@ function recordDocReady() {
     backgroundLoadTab(el.dataset.tab);
   });
 
+  initTruncate('.truncate-subjects', $('.subject-line'));
```

Comment:
As it currently stands, I think the second parameter should be `$('truncate-subjects > .subject-line')` or something like that. However, if the second parameter is always a selector for limiting elements inside the container, how about making it a selector like `'.subject-line'` and then in the actual method do `var target = holder.find(_target);` ?